apps/darts-modernisation/darts-portal/stg.yaml

```diff
@@ -7,5 +7,4 @@ spec:
   releaseName: darts-portal
   values:
     nodejs:
-      ingressHost: darts-portal.staging.platform.hmcts.net
-      
+
```

Comment:
@timja this was here to match the public DNS for staging, otherwise using the environment I was getting `darts-portal.stg.platform.hmcts.net`.

What is the best way to get around that?